CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,24 @@
+# Code of Conduct
+
+The Twin Cities Mutual Aid Map community welcomes participation from everyone. 
+
+## Expectations 
+
+Our community expects that participants:
```

Comment:
What happens when members don't meet these expectations?

Reply:
From the CfA CoC https://brigade.codeforamerica.org/about/code-of-conduct

> If a participant engages in harassing behavior, the organizers may take any action they deem appropriate, including warning the offender or expulsion from Code for America network activities, events, and digital forums.

Reply:
This is a really good point! Thanks for calling that out. I'll add the CfA language.